### DISCUSSION TITLE
`Invoke-FalconRTR` with `put-and-run` and `-CommandLine=`

### DISCUSSION THREAD

**Discussion by swiru95** [CrowdStrike/psfalcon#292] [Q&A] [answered]
When invoking Invoke-FalconRTR the command requires to specify only Executable name instead of additional two arguments:
-CommandLine <commandline str>
-Wait

When passed to -Arguments "Filename.exe -CommandLine abcd..." it thrown 40408 - Put file not found by name.

**Comment by bk-cs** [comment, marked as answer]
Is that the exact command you used? Based on what you're describing, the syntax should look like this:
```
Invoke-FalconRtr -Command put-and-run -Argument "filename.exe -CommandLine='abc123'"
```
I just tested using this syntax and it worked as expected.

**Comment by Xlichtx011** [comment]
I tried to run any exe file in the computer using the command 
Invoke-FalconRtr -Command put-and-run -Argument "filename.exe , but the filename.exe runs on the processes and no window appeared on the screen.

**Reply by bk-cs** [reply]
This is expected behavior. RTR runs processes as system and does not have access to the UI, so you will never see the program visibly displayed inside of an active user session.
